src/earn/EarnHome.tsx

```diff
@@ -238,7 +238,7 @@ export default function EarnHome({ navigation, route }: Props) {
     return pools.filter((pool) => {
       const depositTokenInfo = allTokens[pool.dataProps.depositTokenId]
       const isMyPool = new BigNumber(pool.balance).gt(0) && !!depositTokenInfo
-      return activeTab === EarnTabType.MyPools ? isMyPool : !isMyPool
+      return activeTab === EarnTabType.MyPools ? isMyPool : pools
```

Comment:
I think `EarnTabType` enum also should be update to be AllPools rather than OpenPools just for consistency

Reply:
Updated the enum in 499eb47ad2f48081a3680818340b064f644073e3!